src/components/RideForm/RideForm.js

```diff
@@ -47,10 +96,51 @@ const LabelStyled = styled.label`
   width: 100%;
   display: flex;
   justify-content: space-between;
+  align-items: center;
+`
+const DurationFieldset = styled.fieldset`
+  border: none;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: nowrap;
+  align-items: center;
+`
+
+const DurationLegend = styled.legend`
+  float: left;
 `
 
 const ButtonStyled = styled.button`
```

Comment:
You don't necessarily need to add `Styled` as you don't have another component that is called `Button`. Same for `LabelStyled` and `FormStyled`. It's not wrong and you can do it like this, it's just my opinion :)
```suggestion
const Button = styled.button`
```